rebasehelper/specfile.py

```diff
@@ -102,6 +103,82 @@ def get_strip(self):
         return self.strip
 
 
+class SpecContent(object):
+    """Class representing content of a SPEC file."""
+
+    SECTION_HEADERS = [
+        '%package',
+        '%prep',
+        '%build',
+        '%install',
+        '%check',
+        '%clean',
+        '%prerun',
+        '%postrun',
+        '%pretrans',
+        '%posttrans',
+        '%pre',
+        '%post',
+        '%files',
+        '%changelog',
+        '%description',
+        '%triggerpostun',
+        '%triggerprein',
+        '%triggerun',
+        '%triggerin',
+        '%trigger',
+        '%verifyscript',
+        '%sepolicy',
+        '%filetriggerin',
+        '%filetrigger',
+        '%filetriggerun',
+        '%filetriggerpostun',
+        '%transfiletriggerin',
+        '%transfiletrigger',
+        '%transfiletriggerun',
+        '%transfiletriggerpostun',
+    ]
+
+    def __init__(self, content):
+        self.sections = CaseInsensitiveDict()
+        self._split_sections(content)
```

Comment:
This could be implemented as a class method returning the dict, that way it would be possible to use it even outside this class (not that it's necessary, but it makes a little more sense to me).